client/views/tourview.html

```diff
@@ -423,17 +359,9 @@
                                       </div>
                                     </span>
                                   <img src="../img/moses.jpg" alt="" class="circle">
-                                </li>
+                        </li>
                   </ul>
             </div>
           </div>
         </div>
-        <div class="footer">
-          <p class="text-center">This game is based on the <span class="footer-text">Card Against Humanity</span> card game, which is CC licensed (BY-NC-SA). Cards For Humanity is not affiliated with Cards Against Humanity.</p>
-        </div>
-      </div>
-  <script src="https://code.jquery.com/jquery-3.3.1.min.js"></script>
-  <script src="https://cdnjs.cloudflare.com/ajax/libs/materialize/1.0.0-rc.2/js/materialize.min.js"></script>
-  <script src="../js/gamepage.js"></script>
-</body>
-</html>
\ No newline at end of file
+      </div>
```

Comment:
Just like Victoria would always note. We missed an extra line here.

Reply:
Ok